src/sas/qtgui/Plotting/MaskEditor.py

```diff
@@ -218,7 +218,7 @@ def updateMask(self, mask):
         temp_mask = temp_mask / temp_mask
         temp_mask[mask] = temp_data.data[mask]
 
-        temp_data.data[mask == False] = temp_mask[mask == False]
```

Comment:
This is an array slice. It is looking for an explicit value in the array. This should go back to the original state.